imodqgis/widgets/unique_color_widget.py

```diff
@@ -107,6 +115,37 @@ def classify(self) -> None:
                 Qt.ItemIsEnabled | Qt.ItemIsEditable | Qt.ItemIsSelectable
             )
 
+    def _get_tiled_normalized_midpoints(self, n_elements, tilesize):
+        """
+        Create array with length n_class of rolling normalized midpoint values
+        which can be used to to fetch values on unique colors colormap.
+
+        Example
+        -------
+        >>> self._get_tiled_normalized_midpoints(n_elements=6, tilesize=4)
+        array([0.125, 0.375, 0.625, 0.875, 0.125, 0.375)]
```

Comment:
This example is extremely confusing? Why are the numbers not monotonic?

Reply:
Confusion is caused by bad function naming: _"cyclic"_ was the word I was looking for!